src/KeyManager/MemoryKey/index.ts

```diff
@@ -0,0 +1,44 @@
+import { generateMnemonic, validateMnemonic } from 'bip39'
+import { BlockchainSettings, Bip44AccountPrivate } from 'cardano-wallet'
+import { InvalidMnemonic } from '../errors'
+import { getBindingsForEnvironment } from '../../lib/bindings'
+import { AddressType } from '../../Wallet'
+import { KeyManager } from '../KeyManager'
+const { AccountIndex, AddressKeyIndex, Bip44RootPrivateKey, Entropy, TransactionFinalized, Witness } = getBindingsForEnvironment()
+
+const HARD_DERIVATION_START = 0x80000000
+
+interface MemoryKey extends KeyManager {

```

Comment:
I think we can just conform to the `KeyManager` interface by taking the mnemonic as an argument...